### PR TITLE
jobs: add a job to build snappy on pull requests

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

**Comment by elopio** [comment]
There is one unit test failing. I'll ask Chipaca for help tomorrow.

File: containers/jenkins-master/config/jobs/github-snappy-builddeb/config.xml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+set -ex
+
+. $HOME/.openstack/novarc
+
+export GOPATH=$WORKSPACE
+cd $GOPATH/src/github.com/ubuntu-core/snappy
+
+sudo apt-get update
+sudo apt-get install --yes devscripts equivs
```
Comment (fgimenez) [inline containers/jenkins-master/config/jobs/github-snappy-builddeb/config.xml]: IMO we should add this to the Dockerfile

File: containers/jenkins-master/config/jobs/github-snappy-builddeb/config.xml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  </triggers>
+  <concurrentBuild>true</concurrentBuild>
+  <builders>
+    <hudson.tasks.Shell>
+      <command><![CDATA[
+#!/bin/bash
+
+set -ex
+
+. $HOME/.openstack/novarc
```
Comment (fgimenez) [inline containers/jenkins-master/config/jobs/github-snappy-builddeb/config.xml]: Do we need this?

**Comment by fgimenez** [comment]
Two nits, LGTM

**Comment by elopio** [comment]
Fixed. Thanks for the review Federico.
I'm going to merge this, I'm ok with it showing as failure on the PRs until chipaca fixes it.

**Pull request merged**